cas-server-core/src/main/java/org/jasig/cas/CentralAuthenticationServiceImpl.java

```diff
@@ -218,8 +221,7 @@ public ServiceTicket grantServiceTicket(
                     uniqueTicketIdGenKey);

```

Comment:
I would have expected a bigger cleanup in the `grantServiceTicket` method as I had in mind that most checks were done in the case of proxying...